src/migrations/versions/d066d19614f3_adding_client.py

```diff
@@ -0,0 +1,75 @@
+"""adding_client
+
+Revision ID: d066d19614f3
+Revises: 31387db2a03a
+Create Date: 2022-07-28 09:45:42.057233
+
+"""
+from datetime import datetime
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import Session
+from lib.models.models import User, Client, Order
+
+# revision identifiers, used by Alembic.
+revision = 'd066d19614f3'
+down_revision = '31387db2a03a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Could you just insert it manually? like using a normal `Insert` statement?

Reply:
You can also delete the models file for now.

We can define that later

Reply:
done